sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/Program.cs

```diff
@@ -20,34 +18,19 @@ public static async Task Main(string[] args)
                     Console.WriteLine("[{0:HH:mm:ss:fff}][{1}] {2}", DateTimeOffset.Now, e.Level, message),
                 level: EventLevel.Verbose);
 
-            await ResolveExistingAsync();
-            await TryResolveButNotFoundAsync();
-        }
-
-        private static async Task ResolveExistingAsync()
-        {
-            var dtmi = "dtmi:com:example:TemperatureController;1";
-            var client = new ModelsRepositoryClient();
-
-            IDictionary<string, string> models = await client.ResolveAsync(dtmi).ConfigureAwait(false);
+            // Client init samples
+            ModelResolutionSamples.ClientInitialization();
 
-            Console.WriteLine($"{dtmi} resolved in {models.Count} interfaces.");
-        }
-
-        private static async Task TryResolveButNotFoundAsync()
-        {
-            var dtmi = "dtmi:com:example:NotFound;1";
-            var client = new ModelsRepositoryClient();
+            // Model Resolution samples
+            await ModelResolutionSamples.ResolveExistingModelsFromGlobalRepoAsync();
+            await ModelResolutionSamples.ResolveExistingModelsFromLocalRepoAsync();
+            await ModelResolutionSamples.TryResolveFromGlobalRepoButNotFoundAsync();
+            await ModelResolutionSamples.TryResolveFromLocalRepoButNotFoundAsync();
+            await ModelResolutionSamples.TryResolveWithInvalidDtmi();
```

Comment:
Needs `Async` suffix

Reply:
Done https://github.com/Azure/azure-sdk-for-net/pull/19072